src/reset.css

```
@@ -0,0 +1,4 @@
+body,
+div {
+  margin: 0;
+}
```

div doesn't have default margins
```suggestion
body {
  margin: 0;
}
```